packages/store/addon/-private/system/core-store.ts

```diff
@@ -843,7 +843,8 @@ abstract class CoreStore extends Service {
     has already been assigned an id, you can find the record again using just the lid.
 
     ```app/routes/post.js
-    store.findRecord({ lid });
+    let identifier = recordIdentifierFor(person);
+    store.findRecord({ lid: identifier.lid });
```

Comment:
since you can just pass the identifier directly, I feel demonstrating this in this way is a bit awkward. Until we are able to document accessing identifiers directly from cache (RecordData) apis we should leave how the lid was obtained somewhat ambiguous. Folks using them will have already brushed up on identifier documentation so I think this should be ok.